core/src/main/resources/error/error-classes.json

```diff
@@ -1580,6 +1580,11 @@
       "Not allowed to implement multiple UDF interfaces, UDF class <className>."
     ]
   },
+  "NAMED_ARGUMENTS_SUPPORT_DISABLED" : {
+    "message" : [
+      "Named arguments support is currently disabled. Enable suppport using ALLOW_NAMED_FUNCTION_ARGUMENTS."
```

Comment:
Should write the full name of the configuration: `spark.sql.allowNamedFunctionArguments`.

Reply:
Resolve since it's outdated.